krun.py

```diff
@@ -330,8 +331,4 @@ def attach_log_file(config, resume):
     debug("arguments: %s"  % " ".join(sys.argv[1:]))
     parser = create_arg_parser()
     setup_logging(parser)
-
-    try:
-        main(parser)
-    except:
-        sys.exit(1)
+    main(parser)
```

Comment:
Well, I was thinking that if the exn was caught elsewhere you would say that. So if Krun just bails at this point maybe say `# Krun can fail here if parsing fails` or some such.